### ISSUE TITLE
config file and TRE metric not calculating in evaluation phase for NLST dataset.

### ISSUE THREAD

**Issue by animesh-007** [vigsivan/RWCNet#5]
Hi @vigsivan, I am using the given `train_config.json` for training the model on the NLST dataset.  I also used the eval.py to evaluate the trained model. But in the evaluation phase, eval.py only calculates MSE and not TRE metric.

In the `train_config.json`, values of `dset_min`: `-3024` and `dset_max`: `15622` will be these right for the NLST dataset? I have calculated these values using `get_dset_minmax.py`. Will there be any other change in the config file for the NLST dataset?

Can you please check the `eval.py` for the NLST dataset evaluation?

**Comment by vigsivan** [comment]
I believe the min, max we use for NLST is -4000:4000, but let me get back to you later today.

Re: eval, are you running into issues?

**Comment by animesh-007** [comment]
Yes, in the current version of `eval.py` file it expects `fixed_keypoints` and `moving_keypoints`, which are not present in the data.jon file. Because of which it is not calculating TRE metric for the NLST dataset.

Also, can you share the the min, and maximum value for different datasets used by you?

**Comment by animesh-007** [comment]
Hi, @vigsivan Did you get a chance to look at eval.py for the NLST dataset? Can you please check once if it is working fine?

**Comment by animesh-007** [comment]
Hi @vigsivan. Hope you are doing good. Can you please check how we can calculate the TRE metric with your evaluation code?
